test/e2e/network/netpol/network_legacy.go

```diff
@@ -40,7 +40,7 @@ import (
 	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
 	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
-
+	"k8s.io/kubernetes/test/e2e/network/common"
```

Comment:
what is wrong?
```
test/e2e/network/netpol/network_legacy.go:43:2: imported and not used: "k8s.io/kubernetes/test/e2e/network/common" as network
test/e2e/network/netpol/network_legacy.go:62:9: undefined: common
```

Reply:
This is super strange. But it seems the same was happening in instrumentation and they are importaint as instrumentation there, e.g.:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/instrumentation/logging/generic_soak.go#L30

I will just change it - I'm not going to fight with compiler :)